pco_cli/Cargo.toml

```diff
@@ -20,9 +20,9 @@ anyhow = "1.0.71"
 arrow = { version = "49.0.0", features = ["csv"], default-features = false }
 better_io = { version = "0.1.0", path = "../better_io" }
 blosc-src = { version = "0.3", features = ["zstd"], optional = true }
-bytes = "1.1.0"
+bytes = "1.4.0"
```

Comment:
I understand why we're reducing version requirements, but is there a reason to increase this one?

Reply:
It isn't strictly necessary, but dependency resolution will always choose version 1.4.0+ due to the other dependencies.

Reply:
FYI, I used [cargo-minimal-versions](https://crates.io/crates/cargo-minimal-versions) (with the top-level workspace `Cargo.toml` [removed](https://github.com/taiki-e/cargo-minimal-versions/issues/1)) to check that each `pco` crate builds with [`-Z direct-minimal-versions`](https://crates.io/crates/cargo-minimal-versions).